Harvey2014/Harvey2014.md

```diff
@@ -0,0 +1,16 @@
+# O’Connor 2004
+
+L.D. Danny Harvey. Research Scientist. Dept of Geography, University of Toronto.  2014. Construction of a global disaggregated dataset of building energy use and floor area in 2010. https://doi.org/10.1016/j.enbuild.2014.03.011
```

Comment:
Please correct the author information. It would be nice to have all authors, as used in the existing files.